Nop.Plugin.Shipping.USPS/Services/USPSService.cs

```diff
@@ -492,14 +492,23 @@ private bool IsDomesticRequest(GetShippingOptionRequest getShippingOptionRequest
                 return (shippingOptions, response.Packages.Select(x => $"Error Desc: {x.Error.Description}. USPS Help Context: {x.Error.HelpContext}."));
             }
 
-            shippingOptions.AddRange(response.Packages
-                .SelectMany(x => x.Postage
-                    .Where(isPostageOffered)
-                    .Select(p => new ShippingOption
+            List<int> shippingIdsAdded = new List<int>();
```

Comment:
Packages should be combined into one rate. See also #3

Reply:
I didn’t test it with product config like the bug description. Unsure if USPS api uses the same ID for each rate. I was personally seeing 10+ identical items for shipping internationally, all with ID “2”.

Reply:
Each package have the [unique identifier](https://github.com/nopSolutions/USPS-plugin-for-nopcommerce/blob/nopCommerce-4.20/Nop.Plugin.Shipping.USPS/Services/USPSService.cs#L162). Can you to describe bug?

Reply:
There are multiples of the same package bring returned. All have the same ID.

Reply:
I am not sure how to describe this more clearly... It seems like the USPS api is sending back objects that are duplicate.

Reply:
There is a limit of 70 pounds per package. If the order exceeds 70 pounds, then the USPS request will contain several packages, as well as the USPS response. Each package will have the same weight and price, so it seems that we have duplicate entries, but this is not so. These packages must be combined into one rate. As a result, we must send one order in several packages.

Reply:
We happened onto this bug by accident. Create a product with a high price. Try $1,000,000 (a million) with little or no weight. You will see it behaving oddly.  Honestly this may not even be an issue with nop (as the postal API is the system acting strangely.